modules/Bio/EnsEMBL/Compara/PipeConfig/Fungi/Lastz_conf.pm

```diff
@@ -71,6 +75,23 @@ sub tweak_analyses {
     foreach my $logic_name (keys %overriden_rc_names) {
         $analyses_by_name->{$logic_name}->{'-rc_name'} = $overriden_rc_names{$logic_name};
     }
+
+    # Block unguarded funnel analyses; to be unblocked as needed during pipeline execution.
+    my @unguarded_funnel_analyses = (
+        'check_no_partial_gabs',
+        'update_max_alignment_length_after_FD',
+        'create_alignment_nets_jobs',
+        'remove_inconsistencies_after_chain',
+        'remove_inconsistencies_after_net',
+        'remove_inconsistencies_after_net_fd',
+        'run_healthchecks',
+        'pairaligner_stats',
+        'coding_exon_stats_summary',
```

Comment:
We could reduce this list to the bare minimum, and then remove even `run_healthchecks`, since that is only a funnel analysis in Plants:
```suggestion
        'check_no_partial_gabs',
        'remove_inconsistencies_after_chain',
        'pairaligner_stats',
        'coding_exon_stats_summary',
```